flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/kryo/KryoSerializer.java

```diff
@@ -135,8 +135,7 @@ private static ChillSerializerRegistrar loadFlinkChillPackageRegistrar() {
 
     // ------------------------------------------------------------------------
 
-    private final LinkedHashMap<Class<?>, ExecutionConfig.SerializableSerializer<?>>
-            defaultSerializers;
+    private final LinkedHashMap<Class<?>, SerializableSerializer<?>> defaultSerializers;
```

Comment:
I would prefer not to make this change as it seems inconsequential and unrelated to the scope of this PR.

Reply:
The other changes in this class are similar.